docs/sources/reference/commandline/cli.md

```diff
@@ -1659,6 +1668,19 @@ and linking containers.
 
 #### Examples
 
+    $ sudo docker run --name test -it debian
+    $$ exit 13
+    exit
+    $ echo $?
+    13
+    $ sudo docker ps -a | grep test
+    275c44472aeb        debian:7            "/bin/bash"         26 seconds ago      Exited (13) 17 seconds ago                         test
+
+In this example, we are running `bash` interactivly in the `debian:latest` image, and giving
```

Comment:
interesting, there are many more instances of `bash` in this doc than there are of Bash.

thinking about it :)